site/src/routes/+layout.svelte

```diff
@@ -1,9 +1,8 @@
 <script lang="ts">
-  import "@picocss/pico/css/pico.min.css";
```

Comment:
Kind of a bummer, but pico was causing issues with the tabulator pagination buttons (making them full width). [sakura.css](https://github.com/oxalorg/sakura) looks pretty good though, if not a bit thin on a desktop for showing off visualizations in multi-column fashion.